bigbluebutton-html5/imports/api/presentations/server/methods/exportPresentationToChat.js

```diff
@@ -0,0 +1,56 @@
+import RedisPubSub from '/imports/startup/server/redis';
+import { check } from 'meteor/check';
+import { extractCredentials } from '/imports/api/common/server/helpers';
+import Logger from '/imports/startup/server/logger';
+import setPresentationExporting from '/imports/api/presentations/server/modifiers/setPresentationExporting';
+import Presentations from '/imports/api/presentations';
+
+const EXPORTING_THRESHOLD = 60000;
```

Comment:
There is an open PR that improves the conversion speed, but let's conservatively assume 1 second/slide nonetheless for large slides with high resolution images. BBB's default max is 200 I believe -- tried with one that took 1min18s to convert and the toast disappeared (but the link still showed in chat!)

Reply:
I did a new iteration on it. Maybe you might want to take a look at https://github.com/bigbluebutton/bigbluebutton/pull/15412/commits/07434d238c2138aea7473280c91c52f3054a349e.

Reply:
Awesome, really like that it relies on the number of presentation pages